google-listings-and-ads.php

```diff
@@ -37,11 +37,6 @@
 	return;
 }
 
-// Validate the required versions of everything our plugin depends on.
-if ( ! VersionValidator::instance()->validate() ) {
-	return;
-}
-
```

Comment:
Why was this removed? The VersionValidator is mainly used for confirming the minimum PHP version is met, which we want to do as early as possible. I don't see that check being done anywhere else after the changes.

Reply:
I replaced this with:

```
		// Check requirements.
		if ( ! PluginValidator::validate() ) {
			return;
		}
```

in the plugins_loaded hook

Reply:
added this check back. thanks